public/js/map.js

```diff
@@ -92,102 +93,76 @@ Promise.all([fetchTrpaData, fetchTruckeeData]).then(combinedData => {
     });
 })
 
-
-function createBikeParkingLayer() {
-  const bikeParkingLayerTableId = '1KeuYN2D5EQMkah2TKCrm8gOfTqHCpTuRMi1JwCkK';
-  fetch(`https://www.googleapis.com/fusiontables/v2/query?sql=SELECT%20*%20FROM%20${bikeParkingLayerTableId}&key=${config.googleFusionTablesApiKey}`)
+function createIconLayer(layer, datasetId, style, formattingFunction) {
+  const endpoint = `${MAPBOX_DATASETS_API}/${datasetId}/features?access_token=${config.mapboxAccessToken}`;
+  fetch(endpoint)
   .then((response) => response.json())
-  .then((json) => {
-    if (!json) {
-      error.handleError(new Error('Unable to fetch bike rack data'));
+  .then((geojson) => {
+    if (!geojson) {
+      error.handleError(new Error(`Unable to fetch data for dataset id ${datasetId}`));
       return;
     }
-
-    if (!json.rows) {
-      // No data for this table
-      return;
-    }
-
-    json.rows.forEach((point) => {
-      const bikeRackIcon = L.divIcon({
-        className: 'bike-rack-icon',
-        iconSize: [20, 20],
-      });
-
-      L.marker([parseFloat(point[0]), parseFloat(point[1])], { icon: bikeRackIcon })
-      .addTo(bikeParkingLayer);
+    layer.on('layeradd', (e) => {
+      e.layer.setIcon(L.icon(style));
+      if (formattingFunction) {
+        e.layer.bindPopup(formattingFunction(e.layer.feature.properties));
+      }
     });
+    layer.setGeoJSON(geojson);
   });
 }
 
-function formatBikeShopPopup(shop) {
-  if (shop[1] !== 'LTBC Business Member') {
-    return `<b>${shop[0]}</b>`;
-  }
 
-  const website = shop[5] ? `<a href="${shop[5]}" target="_blank">${shop[5]}</a>` : '';
-  return `<b>${shop[0]}</b><br>LTBC Business Member<br>${shop[2]}<br>${shop[6]}<br>${website}`;
+function createBikeParkingLayer() {
+  const BIKE_PARKING_DATASET_ID = 'ck3gxv6o90p8r2inyo63i5h26';
```

Comment:
Would be nice to group these hard-coded IDs together at the top of the file or in a separate constants file we import from, a-la...

```
const DATASET_IDS = {
  BIKE_PARKING: 'ck3gxv6o90p8r2inyo63i5h26',
  CONSTRUCTION: 'ck3gy4qcm2hg42ol93x46deff',
  ...etc
}
```